BrowserKit/Sources/ToolbarKit/ToolbarManager.swift

```diff
@@ -31,11 +31,11 @@ public class DefaultToolbarManager: ToolbarManager {
         // - the toolbar is displayed at the bottom
         // display the bottom border if
         // - the toolbar is displayed at the top and the website was scrolled
-        // - we are in private mode
+        // - the toolbar is displayed at the top and we are in private mode
         if borderPosition == .top {
             return toolbarPosition == .bottom
         } else {
-            return (toolbarPosition == .top && scrollY > 0) || isPrivate
+            return toolbarPosition == .top && (scrollY > 0 || isPrivate)
```

Comment:
could you elaborate why we are only showing the bottom border if toolbar is on website being scrolled or private mode? At first glance, seems weird to me that we don't show it consistently for both private mode and normal mode and adds additional engineering effort.

Reply:
This was a design decision and the rules for showing the border can be found in the toolbar redesign Figma.